File: trajectory.py
#!/bin/python3
from time import sleep as sleep

speed = 400
tilt = 90
tilt = tilt - 90
if tilt != 0:
    xspeed = 400/(90/tilt)
else: 
    xspeed = 0

yspeed = speed - xspeed

x = 0 
y = 100

for i in range(0,10):
    x += xspeed
    y += yspeed
    print(f"x: {x}. y: {y}")
    sleep(0.5)

File: trajectory.rs
fn main(){
    use std::{thread, time};
    
    let half_second = time::Duration::from_millis(500);
    let speed = 400; // Meters per second
    speed = speed/2;
    let mut tilt = 90;
    let mut xspeed=0;
    let mut yspeed=0;
    let mut x=0;
    let mut y=100;
    tilt = tilt - 90;
    if tilt != 0{
        xspeed = speed/(90/tilt);
    } 
    yspeed = speed - xspeed;
    
    for n in 0..10{
        x += xspeed;
        y += yspeed;
        println!("x: {0} y: {1}", x, y);

        thread::sleep(half_second);
    }
}

